docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,140 @@
+# Kubernetes Deployment
+
+## AnyCable-Go
+
+AnyCable-Go can be easily deployed to your Kubernetes cluster using Helm and [our official Helm chart][anycable-helm].
+
+ 1. Add it as a dependency to your main application:
+
+    ```yaml
+    # Chart.yaml for Helm 3
+    dependencies:
+    - name: anycable-go
+      version: 0.2.4
+      repository: https://helm.anycable.io/
+    ```
+
+    Check the latest Helm chart version at [github.com/anycable/anycable-helm/releases](https://github.com/anycable/anycable-helm/releases).
+
+    And execute
+
+    ```sh
+    helm dependencies update
+    ```
+
+ 1. And then configure it in your application values within `anycable-go` section:
+
+    ```yaml
+    # values.yaml
+
+    # Configuration for the external Helm chart "anycable/anycable-go"
+    anycable-go:
+      env:
+        # Assuming that Ruby RPC is available in K8s in the same namespace as anycable-rpc service (see next chapter)
+        anycableRpcHost: anycable-rpc:50051
+      ingress:
+        enable: true
+        path: /cable
+
+    # values/production.yaml
+    anycable-go:
+      env:
+        # Assuming that Redis is available in K8s in the same namespace as redis-anycable service
+        anycableRedisUrl: redis://:CHANGE-THE-PASSWORD@redis-anycable:6379/0
+        anycableRpcHost: "anycable-rpc:50051" # See service definition for RPC server
```

Comment:
We already have this line in `values.yaml`
```suggestion
```

Reply:
Good catch, thank you!